devices/Sim7080/samples/Program.cs

```diff
@@ -0,0 +1,148 @@
+using IoT.Device.Sim7080;
```

Comment:
For the sample, you need as well a nfproj so it makes it easy for people to use it.
Project reference is the way to go in this case of course.

Reply:
The solution contains 2 projects. One is the library, the other one is the sample project (nfproj). You can execute and flash the sample to your device. So I think its already there?